Sources/core/objects/Interface.cpp

```diff
@@ -86,6 +92,19 @@ Slot<Item>& HumanInterface::GetActiveHand()
         return l_hand_;
 }
 
+void HumanInterface::UpdatePulling(bool isPulling)
+{
+    pulling_ = isPulling;
```

Comment:
I do not think that mixing up various GUI types is good idea, but even in that case you can just set state as empty (or any not existing state) and then button become invisible.